test/Libraries/RevitNodesTests/Elements/GroupTests.cs

```diff
@@ -0,0 +1,164 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Autodesk.DesignScript.Geometry;
+using NUnit.Framework;
+using Revit.Elements;
+using RevitTestServices;
+using RTF.Framework;
+
+namespace RevitNodesTests.Elements
+{
+    [TestFixture]
+    public class GroupTests : RevitNodeTestBase
+    {
+        private const double Tolerance = 0.001;
+
+        [Test]
+        [TestModel(@".\element.rvt")]
+        public void CanCreateGroupFromListOfElements()
+        {
+            // Arrange
+            List<int> elementIds = new List<int>() { 184176, 184324 };
+            List<Element> elementsToGroup = elementIds.Select(id => ElementSelector.ByElementId(id, true)).ToList();
+
+            // Act
+            var group = Group.FromElements(elementsToGroup);
+
+            // Assert
+            Assert.AreEqual(207894, group.Id);
+        }
+
+        [Test]
+        [TestModel(@".\Group\GroupTest.rvt")]
+        public void CanPlaceGroupInstance()
+        {
+            // Arrange
+            Element groupType = ElementSelector.ByElementId(316699, true).ElementType;
+            var expectedLocationForNewGroup = Point.ByCoordinates(0, 0, 0);
```

Comment:
let's not use `0,0,0` since that's the default location for a lot of behaviours, we should check it works for whatever values, like `13,17,19` ( i like primes 🤓 )